.github/workflows/publish.yaml

```
@@ -0,0 +1,25 @@
+name: publish
+on:
+  push:
+
+    branches:    
+      - 'main'
+jobs:
```

Just a whitespace touchup
```suggestion
  push:
    branches:    
      - 'main'

jobs:
```

---

Done!